source/daplink/daplink.sct

```diff
@@ -33,6 +33,8 @@ LR_IROM1 DAPLINK_ROM_APP_START DAPLINK_ROM_APP_SIZE {
   RW_IRAM1 DAPLINK_RAM_APP_START DAPLINK_RAM_APP_SIZE {  ; RW data
    .ANY (ram_func)
    .ANY (+RW +ZI)
+   main.o (+RW)
+   startup_*.o (+ZI)
```

Comment:
Any reason you didn't use `(+RW +ZI)` for both `main.o` and `startup_*.o`? Presumably the excluded type isn't required by the code right now, but it might be in the future and it would be a painful thing to debug.

Reply:
I wanted to pin down only the minimum needed.
I'll give it a go with both RW and ZI.
If it's fine, I'll update this PR.